### PR TITLE
Add magazine wells for WS CDLC

### PR DESCRIPTION
**Pull request by Dahlgren**
**When merged this pull request will:**
- Enable WS weapons to use magazines implementing CBA magazine wells
- Enable WS magazines to be used in other weapons implementing CBA magazine wells

### CODE REVIEW

**Comment by Dahlgren** [comment]
Needs to be tested in game for UBC issues and inconsistencies.

The underbarrel grenade launchers have not been updated with CBA JAM magazine wells.

File: addons/jam/jam_ws/CfgMagazineWells.hpp [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+            "20Rnd_762x51_slr_desert_tracer_green_lxWS",
+            "20Rnd_762x51_slr_reload_tracer_green_lxWS",
+            "20Rnd_762x51_slr_tracer_green_lxWS"
+        };
+    };
+
+    class CBA_762x51_FAL_L {
+        WS_mags[] = {
+            "30Rnd_762x51_slr_lxWS",
+            "30Rnd_762x51_slr_lxWS_desert",
```
Comment (Mike-MF) [inline addons/jam/jam_ws/CfgMagazineWells.hpp:119]:
```suggestion
            "30Rnd_762x51_slr_desert_lxWS",
```

**Comment by Mike-MF** [comment]
These are WS Guns using magazines available via magwells from this PR. Pretty much none of them fit (Only affecting the AA40, SLR & Galil) they'd probably benefit from an alternative magwell.

File: addons/jam/jam_ws/CfgMagazineWells.hpp (hunk 1 above)
Comment (Dahlgren) [reply addons/jam/jam_ws/CfgMagazineWells.hpp:119]: Thanks, I found another such erroneous while updating to new structure but missed this one!

**Comment by Drofseh** [comment]
Having an alternative magwell seems to defeat the point of magwells to me.
I'd rather have magazine compatibility that looks a little ugly than magazines that aren't compatible.

**Review by PabstMirror — APPROVED** [review]

**Comment by jonpas** [comment]
Rifles that don't fit other mags should have magwells removed.

**Review by jonpas — APPROVED** [review]

**Pull request merged**